tests/util/base.py

```diff
@@ -37,6 +38,22 @@ def dbloader(config):
     setup_test_db()
 
 
+@yield_fixture
+def patched_gevent_sleep(monkeypatch):
+    monkeypatch.setattr('gevent.sleep', mock.Mock())
+    yield
+    monkeypatch.undo()
+
+
+@yield_fixture
+def patched_gevent(monkeypatch):
+    monkeypatch.setattr('gevent.sleep', mock.Mock())
+    monkeypatch.setattr('gevent.spawn', mock.Mock())
```

Comment:
Are we able to mock `spawn` and `kill` just because `test_folder_sync.py` doesn't test the functionality of `change_poller`? I think it'd be good to throw a docstring on this fixture to clarify its implications.